README.md

```diff
@@ -152,3 +152,54 @@ This is the template PVC. No namespace is supplied since the PVC is excpected to
 1. Monitor the importer pod:
 
         $ kubectl logs <unique-name-of-importer pod>  # shown in controller log above
+
+### Security Configurations
+
+#### RBAC Roles
+
+CDI needs certain permissions to be able to execute properly, primarily the `cluster-admin` role should be applied to the service account being used through the [Kubernetes RBAC model](https://kubernetes.io/docs/admin/authorization/rbac/).  For example, if CDI is running in a namespace called `golden-images`, and the `default` service account is being used, then the following RBAC should be applied:
+
+```
+  $ kubectl create clusterrolebinding <binding name> --clusterrole=cluster-admin  --serviceaccount=<namespace>:default
+
+  i.e.
+  $ kubectl create clusterrolebinding c-golden-images-default --clusterrole=cluster-admin  --serviceaccount=golden-images:default
+
+```
+
+> NOTE: This gives full cluster-admin access to this binding
+
+
+#### Protecting the `Golden` Image Namespace
+
+Currently there is no support for automatically implementing [Kubernetes ResourceQuotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) and Limits on desired namespaces and resources, therefore administrators need to manually lock down all new namespaces from being able to use the StorageClass associated with CDI/Kubevirt and cloning capabilities. This capability of automatically restricting resources is in the works for a future release. Below are some examples of how one might achieve this level of resource protection:
+
+- Lock Down StorageClass Usage for Namespace:
+
+```
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: protect-mynamespace
+spec:
+  hard:
+    <storageclass name>.storageclass.storage.k8s.io/requests.storage: "0"
```

Comment:
nit: I'd put a dash between _<storageclass_ and _name>_ here, or combine them as _<storageClassName>_. The idea being to not have a space in this token.

Reply:
ack